frappe/integrations/doctype/ldap_settings/ldap_settings.py

```diff
@@ -85,7 +85,7 @@ def sync_roles(self, user, additional_groups=None):
 		needed_roles = set()
 		needed_roles.add(self.default_role)
 
-		lower_groups = [g.lower() for g in additional_groups]
+		lower_groups = [g.lower() for g in additional_groups] if additional_groups != None else []
```

Comment:
```
lower_groups = [g.lower() for g in additional_groups or []]
```
Isn't it better?
Is this the only place where this should be fixed?

Reply:
this looks better. Will change. Yes this one place fixes issue.